addons/mail/models/res_users.py

```diff
@@ -18,6 +18,16 @@ class ResUsers(models.Model):
     """
     _inherit = 'res.users'
 
+    res_role_ids = fields.Many2many(
+        "res.role",
+        string="Res Roles",
+        help="Users are notified whenever one of their roles is @-mentioned in a conversation.",
+    )
+    is_readonly_res_role_ids = fields.Boolean(
```

Comment:
could also drop the "res" from here

But even without that, the name feels a bit strange.